k8s/src/main/scala/io/buoyant/k8s/EndpointsNamer.scala

```diff
@@ -8,6 +8,7 @@ import com.twitter.finagle.util.DefaultTimer
 import com.twitter.util._
 import io.buoyant.k8s.v1._
 import io.buoyant.namer.EnumeratingNamer
+
```

Comment:
I think buoyant style is just to have all imports alphabetically sorted without any breaks like this